docs/user-questions.rst

```diff
@@ -95,6 +95,29 @@ To load both::
     m, t = get_model_and_toas(parfile, timfile)
 
 
+Create TOAs from a list of times
+--------------------------------
+A :class:`pint.toa.TOA` object contains both a time and a location, along with optional information like frequency, measurement error, etc.  
+So each :class:`~pint.toa.TOA` object should only contain a single time, since otherwise the location information would be ambiguous.
+If you wish to create TOAs from a list of times you should use list comprehension to create a list of :class:`~pint.toa.TOA` objects,
```

Comment:
instead of "from a list of times", maybe "from a `Time` object containing an array of times"